crates/notion-core/src/image/mod.rs

```diff
@@ -89,12 +89,15 @@ mod test {
     use std::path::PathBuf;
     use semver::Version;
 
+    #[cfg(unix)]
+    use dirs;
+
     #[cfg(windows)]
     use winfolder;
 
     fn notion_base() -> PathBuf {
         #[cfg(unix)]
-        return PathBuf::from(std::env::home_dir().expect("Could not get home directory")).join(".notion");
+        return PathBuf::from(dirs::home_dir().expect("Could not get home directory")).join(".notion");
```

Comment:
Do we want to have separate `notion_base` and `shim_dir` functions here for these tests? Can we just replace these with the `path::notion_home()` and `path::shim_dir()` functions in the tests?

It seems like with the two separate implementations, the tests could fail if the `NOTION_HOME` environment variable is set to something other than the default, since the actual implementation will use the correct value, while the test will be checking for the default value.

Reply:
I'm not sure I understand this comment? Which tests are you referring to? This code isn't test code. I'm a little confused, sorry!

Reply:
> Doesn't the #[cfg(test)] on line 84 above this indicate that the whole block is test code? 

Yeeeeessss it does and I always forget to look at the collapsed code >_<

Reply:
> so is there an explicit reason why we aren't just using those functions

Now I get it! Great point, I'll fix that.